### ISSUE TITLE
Prevent redundant server instances

### ISSUE THREAD

**Issue by hryx** [leafo/lapis#247]
`lapis server` should do nothing if the server is already running. Or, make a way to check if it is already running.

**Comment by lordnynex** [comment]
@hryx Hello. Lapis does this by checking for nginx pid. This can happen if your openresty build is configured to stick it's pidfile somewhere specific as opposed to the default. In this case, you can define a pidfile location in the nginx.conf template file and restarting.

**Comment by hryx** [comment]
Hey! Thanks for the follow-up. At the time I wrote I didn't have time to be explicit about the problem we encountered (sorry about that). I do have a pidfile location specified in nginx.conf, though.

@leafo Do you remember exactly what the issue was here?

**Comment by leafo** [comment]
Yeah, we wanted an easy to command to either reload the server if it's running, or start it if it's not
